### PR TITLE
Update collect data mode assets

### PR DESCRIPTION
**Pull request by pjanik**
[#184076177]

@kswenson, I'm tagging you as reviewer, as Dough and Lucy are on vacation for a while.

### CODE REVIEW

File: js/plates-view/render-cross-section.ts [hunk 1]
```diff
@@ -337,8 +337,8 @@ class CrossSectionRenderer {
       return;
     }
     // Based on the dropped-pin.png size.
-    const kImgWidth = 21;
-    const kImgHeight = 26;
+    const kImgWidth = 48 * 0.42;
+    const kImgHeight = 72 * 0.42;
```
Comment (kswenson) [inline js/plates-view/render-cross-section.ts:341]: Perhaps a comment or constant explaining the `0.42`?

**Review by kswenson — APPROVED** [review]
👍 Looks good -- I didn't try running the code, but the changes seem straightforward.

File: js/plates-view/render-cross-section.ts (hunk 1 above)
Comment (pjanik) [reply js/plates-view/render-cross-section.ts:341]: Assets are kinda in a draft state. This one was missing, so I had to modify another one by hand, and adjust the size. I guess it'll be soon replaced by the final version.

Comment (pjanik) [reply js/plates-view/render-cross-section.ts:342]: Just pushed a comment, thanks.

**Pull request merged**